stages/02_download_data.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+mkdir -p ./download
+
+while read -r first second third fourth; do
+    wget "$first" "$second" "$third" "$fourth"
+    wait
+done < links.txt
```

Comment:
If the lines are all `wget` lines, the contents of `links.txt` are essentially already a shell script.